docs/symphony.md

```diff
@@ -79,14 +79,3 @@ spec:
             name: a-different-input
             namespace: default
 ```
-
```

Comment:
That's interesting... I've been of the mind that the docs should be based on how setting various properties alters the behavior of Eno. So documenting behavior that isn't part of the API isn't really a thing yet. I'd say let's make a note to document this at some point, but I don't think it really fits anywhere currently - cool?

Reply:
Totally cool. I also don't like the idea of having a catch-all file that can potentially (more like surely?) grow unbounded. However, I believe it's worth at least mentioning it since it's a behavior affecting the whole cluster that wouldn't exist otherwise if Eno wasn't running